src/utils/lib.ts

```diff
@@ -45,6 +45,133 @@ export const getLibraryMeta = (
     };
 };
 
+export const getComponentMeta = async (
+    lib: {id: string; title: string},
+    componentTitle: string,
+    componentId: string,
+    locale = 'en',
+    fallbackDescription?: string,
+): Promise<MetaProps> => {
+    let staticDescription = '';
+
+    try {
+        const fs = require('fs');
+        const path = require('path');
+        const filePath = path.join(
+            process.cwd(),
+            'public',
+            'locales',
+            locale,
+            'component-meta.json',
+        );
+        const fileContent = fs.readFileSync(filePath, 'utf8');
```

Comment:
Oh no) We should not use reading files by path, especially at runtime. Use `t` function like in `getLibraryMeta`.